DQM/SiStripMonitorClient/plugins/SiStripOfflineDQM.cc

```diff
@@ -78,7 +78,7 @@ SiStripOfflineDQM::SiStripOfflineDQM(edm::ParameterSet const& pSet)
       auto token = useSSQ ? QualityToken{esConsumes<edm::Transition::EndRun>(
                                 edm::ESInputTag{"", tkMapPSet.getUntrackedParameter<std::string>("ssqLabel", "")})}
                           : QualityToken{};
-      tkMapOptions_.emplace_back(map_type, std::move(tkMapPSet), useSSQ, std::move(token));
+      tkMapOptions_.emplace_back(map_type, std::move(tkMapPSet), useSSQ, token);
```

Comment:
This might be related to https://github.com/llvm/llvm-project/issues/86404 . This might be fixed in llvm 19 , so I can try to update llvm 19 for clang IBs. For now I can revert this change and suppress the clang-tidy warning by adding //NOLINT for this line